tests/TestBase.js

```diff
@@ -0,0 +1,14 @@
+const puppeteer = require('puppeteer');
+
+module.exports = class TestBase {
+  constructor(){
+    this.browser = ""
+  }
+  async createPage(){
+    this.browser = await puppeteer.launch();     
+    return await this.browser.newPage();
+  }  
+  async cleanup(){
```

Comment:
integrating eslint

Reply:
Please ignore this for now. I will ask you to open separate PR after we merge this PR.